packages/node/src/__tests__/integration.test.ts

```diff
@@ -247,7 +247,9 @@ describe('track', () => {
     const track = resolveCtx(analytics, 'track')
     analytics.track({ event: 'hello', userId: 'foo' })
     await track
-    expect(makeReqSpy.mock.calls[0][0].data.sentAt).toBeInstanceOf(Date)
+    expect(
+      new Date(JSON.parse(makeReqSpy.mock.calls[0][0].body).sentAt)
```

Comment:
I don't _think_ this actually throws an error, so it won't fail if date is incorrect.
e.g.
<img width="366" alt="image" src="https://github.com/segmentio/analytics-next/assets/5115498/c602490b-b40d-45c8-904a-be1a0a95fa7f">

can we do:
```ts
import { isDate } from 'lodash'
...
expect(isDate(JSON.parse(makeReqSpy.mock.calls[0][0].body).sentAt)).toBeTruthy()
```